tools/bazel_build_cache/setup_ci_build_cache.sh

```diff
@@ -3,15 +3,6 @@
 # The script should immediately exit if any command in the script fails.
 set -e
 
-PULL_BRANCH_REGEX="^pull\/[0-9]+$"
-
-# Forked pull requests have CIRCLE_BRANCH set to pull/XXX
-if [[ "${CIRCLE_BRANCH}" =~ $PULL_BRANCH_REGEX ]]; then
-  echo "Forked pull request no setup for upload to remote chache!"
-  echo "build --remote_upload_local_results=false" >> ../../.circleci/bazel.rc
-  exit 0
-fi
-
```

Comment:
Why do we remove this? It should run the bazel targets on forks as well only without the remote build cache population. As for pushing to the remote build cache a salt is needed and we cannot pass environment variables to forks.